libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.ts

```diff
@@ -118,6 +118,12 @@ export class GridListItemComponent<T> implements OnChanges, AfterViewInit, OnDes
     @Input()
     isNavigated = false;
 
+    /** @hidden
+     * The active state of the list item.
+     * If set to true, the whole card has active state. Becomes false only when the Edit button is clicked.
+     */
+    _isActive = true;
```

Comment:
only private properties should have _ prefix

Reply:
Would not agree, methods without underscore at the start are **public**, so the user can use them to work with components programmatically and all others should be started with an underscore. But, of course, some of them are public in the meaning of the component (used in the template). 

What do you think?

Reply:
agree with @platon-rov 
and it is already discussed and approved but not updated the wiki

Reply:
@Betrozov @platon-rov If we use private variable inside template we will get error during AOT compilation. Angular disallow using private vars inside template. We have Code Contribute Guide which [tells](https://github.com/SAP/fundamental-ngx/wiki/Code-Contribution-Guideline#method-and-variable-naming-convention) that we should use underscore as prefix for private vars.We shouldn't use underscore as a prefix for public variables.

Reply:
@Smiranin
I'm talking about using `_` prefix for private variables and for variables that should be used in template but hidden from outside

Reply:
I had talk with Yury and as I figure out there is one addition rule related to @hidden variables: "use underscore prefix for @hidden variable". 
In this case need to update Code Contribute Guide.

**But,** In my opinion, it is better to add an new prefix for hidden variables if they need to be labeled in a special way. And do not interfere with them with private ones, because this is confusing.